contracts/discovery/GNS.sol

```diff
@@ -286,6 +289,10 @@ contract GNS is GNSV1Storage, GraphUpgradeable, IGNS, Multicall {
             "GNS: Cannot publish a new version with the same subgraph deployment ID"
         );
 
+        NameCurationPool storage namePool = nameSignals[_graphAccount][_subgraphNumber];
+
+        curation().setCreatedAt(_subgraphDeploymentID, namePool.createdAt);
```

Comment:
I don't feel comfortable with this GNS->Curation interaction that makes Curation completely dependent on the GNS, but I'm trying to think if there is a better option and couldn't think of anything better yet.